src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -2,6 +2,69 @@
 
 public class SalaryInfo {
     public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+        int salaryFirst = 0;
```

Comment:
Any magic numbers should be constants. The initial salary should be defined as a constant to improve readability and maintainability.